docs/building.md

```diff
@@ -120,33 +120,6 @@ the registry binary generated in the "./bin" directory:
     $ ./bin/registry -version
     ./bin/registry github.com/docker/distribution v2.0.0-alpha.2-80-g16d8b2c.m
 
-### Developing
-
-The above approaches are helpful for small experimentation. If more complex
-tasks are at hand, it is recommended to employ the full power of `godep`.
-
-The Makefile is designed to have its `GOPATH` defined externally. This allows
```

Comment:
This statement should not have been removed. This declares the intent of the Makefile in this project. The parts about `godep` can be removed, but the idea that GOPATH is external to the Makefile is still valid.